clojure-mode.el

```diff
@@ -329,6 +329,33 @@ instead of to `clojure-mode-map'."
   (clojure-font-lock-setup)
   (add-hook 'paredit-mode-hook #'clojure-paredit-setup))
 
+(defcustom clojure-do-warnings t
```

Comment:
I'd name this differently - e.g. `clojure-verify-major-mode`. How do you even do warnings? :-)